packages/router/src/router.ts

```diff
@@ -221,7 +221,7 @@ class Router<RouteRecord extends IRouteRecord> implements IRouter<RouteRecord> {
       onAbort && onAbort();
 
       // fire ready cbs once
-      if (!this._ready) {
+      if (!this._ready && this._current !== START) {
```

Comment:
avoid unexpected router ready in abort

this fixes https://github.com/shuvijs/shuvi/pull/561